gradle.properties

```diff
@@ -25,4 +25,6 @@ android.enableR8.fullMode=true
 org.gradle.unsafe.configuration-cache=false
 # PubNub keys
 PUBNUB_PUBLISH_KEY="myPublishKey"
-PUBNUB_SUBSCRIBE_KEY="mySubscribeKey"
\ No newline at end of file
+PUBNUB_SUBSCRIBE_KEY="mySubscribeKey"
+# YouTube key
+YOUTUBE_KEY="muYoutubeKey"
```

Comment:
Add the link https://developers.google.com/youtube/android/player/register to readme?